CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+This changlog uses the [ISO 8601 date format](https://www.iso.org/iso-8601-date-and-time-format.html) of (YYYY-MM-DD).
```

Comment:
There is a typo: changlog instead of changelog.

Reply:
@sean-hut ^

Reply:
I personally learned more. The [RFC 3339](https://www.rfc-editor.org/rfc/rfc3339#section-5.6) defines a profile for ISO 8601.

I would propose:

```markdown
This changelog uses the `full-date` format of [RFC 3339](https://www.rfc-editor.org/rfc/rfc3339#section-5.6), which is `YYYY-MM-DD`.
```

Reply:
@koppor while your proposal would be technically more precise, I feel it's an overkill due to unnecessary complexity.

Just open your referenced link. Does one really need to know all these semantics to understand that a date (a "full date" that is) is represented as simple as `YYYY-MM-DD`?